src/Jellyfin.Extensions/Json/JsonDefaults.cs

```diff
@@ -36,6 +36,7 @@ public static class JsonDefaults
                 new JsonGuidConverter(),
                 new JsonNullableGuidConverter(),
                 new JsonVersionConverter(),
+                new JsonFlagEnumConverterFactory(),
```

Comment:
Does this work as expected? When is `JsonStringEnumConverter` used now?

Reply:
`JsonStringEnumConverter` is used for any enum that is not `[Flags]`